### PR TITLE
Add components for floating map logo and scale combo

### PR DESCRIPTION
**Pull request by ahennr**
Introduces `FloatingMapLogo` and `ScaleCombo` components.

In addition, `MapUtil` is included here as well beside method `buildLayersFromInitialState` (see https://github.com/terrestris/react-geo/pull/12)

### CODE REVIEW

File: src/Map/ScaleCombo/ScaleCombo.jsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    };
+  }
+
+  /**
+   * Unknown - Description
+   *
+   * @param {type} resolution Description
+   * @param {type} mv         Description
+   *
+   * @return {type} Description
```
Comment (dnlkoch) [inline src/Map/ScaleCombo/ScaleCombo.jsx]: Wrong JSDoc.

File: src/Map/ScaleCombo/ScaleCombo.jsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    this.props.scales.push(option);
+  };
+
+  /**
+   * Unknown - Description
+   *
+   * @param {type} resolution Description
+   * @param {type} mv         Description
+   *
+   * @return {type} Description
```
Comment (dnlkoch) [inline src/Map/ScaleCombo/ScaleCombo.jsx]: Wrong JSDoc.

File: src/Map/FloatingMapLogo/FloatingMapLogo.example.jsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render } from 'react-dom';
+import FloatingMapLogo from './FloatingMapLogo.jsx'; //@react-geo@
+
+render(
+  <FloatingMapLogo imageSrc="logo.png"/>,
```
Comment (dnlkoch) [inline src/Map/FloatingMapLogo/FloatingMapLogo.example.jsx]: Can we make use of a image-loader here?

File: src/Map/ScaleCombo/ScaleCombo.jsx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+   */
+  pushScaleOption = (resolution, mv) => {
+    let scale = MapUtils.getScaleForResolution(resolution, mv.getProjection().getUnits());
+    let roundScale = Math.round(scale);
+    let option = <Option key={roundScale.toString()} value={roundScale.toString()}>1:{roundScale.toLocaleString()}</Option>;
+    this.props.scales.push(option);
```
Comment (dnlkoch) [inline src/Map/ScaleCombo/ScaleCombo.jsx]: Props are immutable inside the component. Please make use of the state instead.

File: src/Map/ScaleCombo/ScaleCombo.jsx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+  };
+
+  /**
+   * Determine option element for provided zoom level out of array of valid options.
+   *
+   * @param {Number} zoom zoom level
+   *
+   * @return {Element} Option element for provided zoom level
+   */
+  getVal (zoom) {
```
Comment (dnlkoch) [inline src/Map/ScaleCombo/ScaleCombo.jsx]: Can we rename this to something more meaningful?

**Comment by dnlkoch** [comment]
Top! Great addition!

File: src/Map/ScaleCombo/ScaleCombo.jsx (hunk 1 above)
Comment (ahennr) [reply src/Map/ScaleCombo/ScaleCombo.jsx]: Fixed in bef0bbd

File: src/Map/ScaleCombo/ScaleCombo.jsx (hunk 4 above)
Comment (ahennr) [reply src/Map/ScaleCombo/ScaleCombo.jsx]: Fixed in bef0bbd

File: src/Map/ScaleCombo/ScaleCombo.jsx (hunk 2 above)
Comment (ahennr) [reply src/Map/ScaleCombo/ScaleCombo.jsx]: Fixed in bef0bbd

File: src/Map/ScaleCombo/ScaleCombo.jsx (hunk 5 above)
Comment (ahennr) [reply src/Map/ScaleCombo/ScaleCombo.jsx]: Renamed to `determineOptionKeyForZoomLevel` in bef0bbd

File: src/Map/ScaleCombo/ScaleCombo.jsx (hunk 1 above)
Comment (ahennr) [reply src/Map/ScaleCombo/ScaleCombo.jsx]: Fixed in bef0bbd

File: src/Map/FloatingMapLogo/FloatingMapLogo.example.jsx (hunk 3 above)
Comment (ahennr) [reply src/Map/FloatingMapLogo/FloatingMapLogo.example.jsx]: Alright, use image loader in 0607940

**Comment by ahennr** [comment]
Thx for review @dnlkoch . Will merge as soon as all checks are green.

**Pull request merged**